### PR TITLE
Core refactor

### PR DESCRIPTION
**Pull request by PuruVJ**
## Description
- [x] Moved `twitter` folder to `twitter-core`.
- [x] Changed printing from A4 to A5 printing
- [ ] Render images, retweets, links(*Work in progress*)
- [x] Higher resolution user profile picture

## Attach Screenshot
image

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by PuruVJ** [comment]
Haven't yet deleted twitter folder cuz wanna have it confirmed by other people. Will delete when confirmed by @Mira-Alf @tolgaerdonmez @johnjacobkenny

**Comment by proful** [comment]
Trusting you @PuruVJ I am merging and all test it out from main branch once.

File: twindle-cli/utils/send-email.js [hunk 1]
```diff
@@ -2,25 +2,30 @@ const nodemailer = require("nodemailer");
...
-	console.devLog("Email sent: %s", info.messageId);
+  // send mail with defined transport object
+  const info = await transporter.sendMail({
+    from: emailConfig.auth.user, // sender address
+    to: emailTo, // list of receivers
+    subject, // Subject line
+    attachments: [
+			{
+				
+			},
```
Comment (johnjacobkenny) [inline twindle-cli/utils/send-email.js:18]: @PuruVJ why is this added

Comment (PuruVJ) [reply twindle-cli/utils/send-email.js:18]: Just some testing. Irrelevant. Will remove

Comment (johnjacobkenny) [reply twindle-cli/utils/send-email.js:18]: Cool

**Comment by johnjacobkenny** [comment]
Also, @PuruVJ isn't twindle the entire process of fetching tweets and then rendering it? What are your thoughts in separating just the twitter part into twindle-core

**Comment by PuruVJ** [comment]
API for our serverless backend

**Comment by PuruVJ** [comment]
Also to be able to run automated tests

**Comment by proful** [comment]
@PuruVJ we no need to work on serverless backend now. Please focus on cli part with out worrying about backend please.
